service/frontend/workflowHandler.go

```diff
@@ -863,6 +864,9 @@ func (wh *WorkflowHandler) PollWorkflowTaskQueue(ctx context.Context, request *w
 	if err != nil {
 		return nil, wh.error(err, scope, tagsForErrorLog...)
 	}
+	if resp == nil {
+		resp = nilResponse
+	}
```

Comment:
This shouldn't happen, right? `wh.createPollWorkflowTaskQueueResponse` never returns `nil, nil`.